owsi-core/owsi-core-components/owsi-core-component-jpa-more/src/main/java/fr/openwide/core/jpa/more/business/difference/model/Difference.java

```diff
@@ -0,0 +1,41 @@
+package fr.openwide.core.jpa.more.business.difference.model;
+
+import de.danielbechler.diff.node.DiffNode;
+import de.danielbechler.diff.path.NodePath;
+
+public class Difference<T> {
+	
+	private final T before;
+
+	private final T after;
+	
+	private final DiffNode diffNode;
```

Comment:
It would be nice to get rid of this (only) adherence to java-object-diff API. Maybe we should see why this node is (sometimes) needed, and add methods accordingly.